integration-tests/docker/environment-configs/indexer

```diff
@@ -36,3 +36,4 @@ druid_auth_basic_common_cacheDirectory=/tmp/authCache/indexer
 druid_startup_logging_logProperties=true
 druid_server_https_crlPath=/tls/revocations.crl
 druid_worker_capacity=10
+druid_worker_intermediaryPartitionTimeout=PT60M
```

Comment:
did you notice a failure in indexer as well? I don't think it's a problem there since there is only one LocalIntermediaryDataManager instance.

Reply:
I was not sure about indexer. Let me try without it.